subdomains/blog/_posts/2020-11-25-command-spotlight-volta-run.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "Command Spotlight: volta run"
+date:   2020-11-23 08:00:00 -0700
+categories: [spotlight, commands]
+author: Chuck Pierce
+excerpt_separator: <!--more-->
+---
+
+Volta is designed to be invisible: You type a command like `npm start`, and Volta automatically detects and runs the right version of npm. While that works great most of the time, occasionally you need direct control over the tool versions that are used. Maybe you are trying to reproduce reported bug or testing how your project works with a different npm version. Whatever the reason, **`volta run` lets you take control of your tool selection right from the command line**.
```

Comment:
- "right version of npm" ==> how about "right versions of npm and Node"? Just to give a hint that it's not just about npm, but without falling down the slippery slope to completeness of documentation.
- "reproduce reported bug" ==> "reproduce a reported bug"

Reply:
I feel like the bold is a little disproportionately excited for the topic. The "spotlight" series is an introduction to various individual features, but not necessarily trying to advertise the biggest takeaways of Volta, so I think it warrants keeping the tone a little muted.

How about:

> Whatever the reason, `Volta run` lets you **take explicit control of tool selection,** right from the command-line.

Not much different, just slightly toned down. WDYT?

Reply:
I like all those :+1: